app/src/main/java/com/framgia/quangtran/music_42/ui/genre/GenreTrackActivity.java

```diff
@@ -0,0 +1,64 @@
+package com.framgia.quangtran.music_42.ui.genre;
+
+import android.content.ContentResolver;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.framgia.quangtran.music_42.R;
+import com.framgia.quangtran.music_42.data.model.Genre;
+import com.framgia.quangtran.music_42.data.model.Track;
+import com.framgia.quangtran.music_42.data.repository.TrackRepository;
+import com.framgia.quangtran.music_42.data.source.local.TrackLocalDataSource;
+import com.framgia.quangtran.music_42.data.source.remote.TrackRemoteDataSource;
+import com.framgia.quangtran.music_42.util.StringUtil;
+
+import java.util.List;
+
+public class GenreTrackActivity extends AppCompatActivity implements GenreTrackContract.View {
```

Comment:
```suggestion
public class GenreActivity extends AppCompatActivity implements GenreTrackContract.View {
```
tương tự với Adapter, Contract và Presenter